integration_test/functions/src/index.ts

```diff
@@ -150,8 +151,6 @@ export const integrationTests: any = functions
         .storage()
         .bucket()
         .upload('/tmp/' + testId + '.txt'),
-      // Invoke a callable HTTPS trigger.
-      callHttpsTrigger('callableTests', { foo: 'bar', testId }),
```

Comment:
Was this call just not needed? Trying to understand why it was removed and not modified to include three fields.

Reply:
We only need to invoke this once, which we do earlier in the file. I think this was the result of bad merge conflict resolution by git.

It looks like when the merge between public and private happened, this line was moved in one branch and modified to the new signature in the other, so git left both changes in the merge.

Reply:
Hm...that sounds odd that this happened. Also we should always run the integration tests prior to releasing to avoid this in the future.

Reply:
Yeah, this one is on me - i ran them before the merge and must have forgotten to rerun afterwards